README.md

```diff
@@ -854,6 +855,139 @@ registration: {
 ```
 - **registration.click** - Function that is called when the registration button is clicked
 
+
+## Registration
+
```

Comment:
Can we add an introduction here before jumping straight into the code block?

Reply:
@ahmedalbahar-okta Can we get some intro text to add to the docs for Registration ?

Reply:
@jmelberg-okta / @nikhilvenkatraman-okta 
"Registration 

Feature flag with additional configuration in your Okta admin settings to allow users to self service register into your app."

I parsed through language currently used in the readme and the above intro seems consistent . 

@mystiberry-okta any thoughts?

Reply:
`Use Okta Sign-In Widget with registration feature and additional configuration in your Okta admin settings to allow users to self service register into your app.`

How about this ?

Reply:
This sounds like you only have to configure it in the admin settings in your Okta tenant.

Suggestion:
To add registration into your application, configure your Okta admin settings to allow users to self register into your app. Then, add the `registration` key to the [`OktaSignIn`](#new-oktasigninconfig) object.

Reply:
@jmelberg-okta There is a piece that needs to be configured in the Okta admin side. You need to enable the feature and turn on registration for you app

Reply:
oh, nvm i misunderstood your comment. I am ok with this text.

Reply:
How is this ?
To add registration into your application, configure your Okta admin settings to allow users to self register into your app. Then, set features.registration in the widget. You can add additional configs under the registration key on the [`OktaSignIn`](#new-oktasigninconfig) object.

Reply:
👍  -> Just make sure to include `features.registration` in back-ticks so it stands out.

Reply:
Done. Will merge once travis is green